src/handlers/swap_handlers.ts

```diff
@@ -166,6 +169,8 @@ const parseGetSwapQuoteRequestParams = (req: express.Request): GetSwapQuoteReque
             ? undefined
             : parseStringArrForERC20BridgeSources(req.query.excludedSources.split(','));
     const affiliateAddress = req.query.affiliateAddress;
+    // tslint:disable-next-line:boolean-naming
+    const intentOnFilling = req.query.intentOnFilling === undefined ? false : true;
```

Comment:
`'false'` would come out as "true" here.  Enforce a specific value, eg "true".

Reply:
Addressed in https://github.com/0xProject/0x-api/pull/162/commits/cbecfc4fd6b0d8df190e8f86cf235fc08aad714c